EDDiscovery/UserControls/Helpers/ScanDisplay.cs

```diff
@@ -350,7 +350,8 @@ Point CreatePlanetTree(List<PictureBoxHotspot.ImageElement> pc, StarScan.ScanNod
                             if (ShowOverlays)
                             {
                                 bool valuable = sc.EstimatedValue > 50000;
-                                int overlaystotal = (sc.Terraformable ? 1 : 0) + (sc.HasMeaningfulVolcanism ? 1 : 0) + (valuable ? 1 : 0);
+                                bool mapped = sc.IsMapped(hl);
```

Comment:
#2263 added a `IsMapped` flag to the scannode.

Reply:
missed that completely... Will change shortly.